perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -4072,5 +4072,48 @@ public GraphDTO call(ApiCaller ac, Deserializer parms) throws PrivilegeException
 
 			return ac.getAttributesManager().getModulesDependenciesGraph(ac.getSession(), format);
 		}
+	},
+
+	/*#
+	 * Marks the action on attribute as critical, which may require additional authentication of user
+	 * performing that action on attribute.
+	 *
+	 * @param sess session
+	 * @param attr attribute definition
```

Comment:
and maybe I would also use the same parameter names for `setAttributeActionCriticality` and  `getAttributeRules` here and in openapi, since we're referring to attribute definition ids in both methods, no?